core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestUtils.java

```diff
@@ -246,6 +249,8 @@ public static S3Exception toObjectS3Exception(Exception exception, String resour
       return new S3Exception(e, resource, S3ErrorCode.NO_SUCH_KEY);
     } catch (AccessControlException e) {
       return new S3Exception(e, resource, S3ErrorCode.ACCESS_DENIED_ERROR);
+    } catch (InvalidPathException e) {
+      return new S3Exception(e, resource, S3ErrorCode.INVALID_BUCKET_NAME);
```

Comment:
why the invalidPathException will be an INVALID_BUCKET_NAME error code in object case? What if the object path is invalid?